x-pack/plugins/infra/public/alerting/log_threshold/components/alert_dropdown.tsx

```diff
@@ -6,12 +6,34 @@
  */
 
 import React, { useState, useCallback, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
 import { EuiPopover, EuiButtonEmpty, EuiContextMenuItem, EuiContextMenuPanel } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { AlertFlyout } from './alert_flyout';
 import { useLinkProps } from '../../../hooks/use_link_props';
+import { useKibanaContextForPlugin } from '../../../hooks/use_kibana';
+
+const readOnlyUserTooltipContent = i18n.translate(
+  'xpack.infra.logs.alertDropdown.readOnlyCreateAlertContent',
+  {
+    defaultMessage: 'Unable to create an alert',
```

Comment:
Nit: can this provide more insight to the user about how to enable the action?

```suggestion
    defaultMessage: 'Creating alerts requires more permissions in this application.',
```